### DISCUSSION TITLE
404 Error in Setup CI (.github#L1 Not Found)

### DISCUSSION THREAD

**Discussion by beaulife** [upptime/upptime#43] [Q&A] [answered]
I carried out Setup CI but received the error below:
   Check failure on line 1 in .github
   @github-actions github-actions / Setup Upptime
   .github#L1
   Not Found

Which part did I do wrong? Looking forward to your reply.

**Comment by AnandChowdhary** [comment]
Can you send me a link to your workflow?

**Comment by beaulife** [comment]
Here it is https://github.com/beaulife/aspire-idea-hosting-uptime/actions/runs/335209074/workflow

**Comment by AnandChowdhary** [comment]
It seems like this is a private repository. Could you please make it public or give me collaborator access?

**Comment by beaulife** [comment]
Yes, already invited you as collaborator to access.

**Comment by AnandChowdhary** [comment]
It looks like accessing the GitHub API got an error 404. Have you added the GitHub personal access token? https://upptime.js.org/docs/get-started#add-repository-secrets

**Comment by AnandChowdhary** [comment, marked as answer]
Aah, I see the error: https://github.com/beaulife/aspire-idea-hosting-uptime/blob/master/.upptimerc.yml#L1.

You have to change the `owner` to "beaulife" and `repo` to "aspire-idea-hosting-uptime". You should also remove the default `notifications` key, `assignees` key, and remove the `cname` from status-website.

See: https://upptime.js.org/docs/get-started
